tests/hawtio-test-suite/src/main/java/io/hawt/tests/features/stepdefinitions/throttling/ThrottlingStepDefs.java

```diff
@@ -0,0 +1,49 @@
+package io.hawt.tests.features.stepdefinitions.throttling;
+
+import com.codeborne.selenide.SelenideElement;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import io.hawt.tests.features.pageobjects.pages.LoginPage;
+import io.hawt.tests.features.setup.LoginLogout;
+
+import static com.codeborne.selenide.Condition.text;
+import static com.codeborne.selenide.Selenide.$;
+
+public class ThrottlingStepDefs {
+    private final LoginPage loginPage = new LoginPage();
+    private final static SelenideElement warning = $("p.pf-c-form__helper-text.pf-m-error");
```

Comment:
It's preferable for this to be just By with the selector and you call the selenide function `$` when you need to access the element. This is because sometimes the elements on the webpage can change and leads to Selenide throwing an exception that the reference to the element is stale, it's rare but annoying :smile_cat: 

Also it's best practice to name constants in `ALL_UPPERCASE`, in this case warning is a constant.